### ISSUE TITLE
Misinterprets tox py{35, 36, 37, 38, py3} as 3.5

### ISSUE THREAD

**Issue by hugovk** [mgedmin/check-python-versions#18]
```console
$ git clone https://github.com/jmoiron/humanize
...
$ cd humanize
$ check-python-versions .
setup.py says:              3.5, 3.6, 3.7, 3.8, PyPy
- python_requires says:     3.5, 3.6, 3.7, 3.8
tox.ini says:               3.5
.travis.yml says:           3.10-dev, 3.5, 3.6, 3.7, 3.8, 3.9-dev, PyPy3

mismatch!
$ cat tox.ini
[tox]
envlist =
    py{35, 36, 37, 38, py3}
...
```

It's interpreting `envlist = py{35, 36, 37, 38, py3}` as only Python 3.5.

This sort of "generative envlist" is fairly common to compress the list, especially with a matrix:

* https://tox.readthedocs.io/en/latest/example/basic.html#compressing-dependency-matrix

**Comment by mgedmin** [comment]
Ooh, thank you or this excellent example!

(Apologies for late reply, I was on vacation and trying to pretend computers didn't exist at all)

**Comment by mgedmin** [comment]
Fix released in check-python-versions 0.15.2.

**Comment by hugovk** [comment]
Thanks!

Pretending computers don't exist on holiday is a great idea!
